internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -479,6 +479,13 @@ func resourceCosmosDbAccount() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			// The `minimalTlsVersion` of the source CosmosDB Account is `Tls12` when it isn't set in the request payload but the `minimalTlsVersion` of the restored CosmosDB Account is `Tls` when it isn't set in the request payload. So the default value isn't added in the schema.
```

Comment:
This would be a breaking change, and users would have to update all their configs. It sounds like the API has a bug and should consolidate on a single default value here? Can you take this up with the service team?

Reply:
@jackofallops , thanks for the suggestion. I will file ticket for minimal_tls_version and I removed it from this PR. Please continue to review this PR for new property partition_merge_enabled. Thanks.